context/context.go

```diff
@@ -66,6 +66,25 @@ func GetOpenFunctionContext() (*Context, error) {
 		}
 	}
 
+	if ctx.PluginsTracing != nil && ctx.PluginsTracing.Enable {
+		if ctx.PluginsTracing.Provider != nil && ctx.PluginsTracing.Provider.Name != "" {
+			switch ctx.PluginsTracing.Provider.Name {
+			case TracingProviderSkywalking, TracingProviderOpentelemetry:
+				ctx.PrePlugins = registerTracingPluginIntoPrePlugins(ctx.PrePlugins, ctx.PluginsTracing.Provider.Name)
+				ctx.PostPlugins = registerTracingPluginIntoPostPlugins(ctx.PostPlugins, ctx.PluginsTracing.Provider.Name)
+			default:
+				return nil, fmt.Errorf("invalid tracing provider name: %s", ctx.PluginsTracing.Provider.Name)
+			}
+			if ctx.PluginsTracing.Tags != nil {
+				if funcName, ok := ctx.PluginsTracing.Tags["func"]; !ok || funcName != ctx.Name {
```

Comment:
we might need to add pod's name to tracing plugin's tags map like `replica: <podname>`, together with `func: <functionname>`, it will provide more accurate tracing data. Simply add `func: <functionname>` is not ok, because a function has many replicas and the tracing should be specific to replica. Maybe the `replica: <podname> should be added in the tracing plugin code, but can we provide a function somewhere to retrieve current function replica's pod name

Reply:
the pod's name can be passed to the container via env.
like this: https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/#use-pod-fields-as-values-for-environment-variables ?

Reply:
yes, that's what I mean

Reply:
The namespace is also useful to be added to tags

Reply:
Added parsing function for the `POD_NAME` and `POD_NAMESPACE` environment variables, which have the following association with `tags` in the Tracing configuration:
```
tags['replica'] = <value from POD_NAME>
tags['namespace'] = <value from POD_NAMESPACE>
```
I wonder if this could solve the problem?

Reply:
As in discussion https://github.com/OpenFunction/functions-framework/issues/9#issuecomment-1003239665, skywalking has `service`, `instance` and `endpoint` concept, for now, it seems that function is a `service`, `replica` is an `instance`, can we change `replica` to `instance`?

Reply:
> Added parsing function for the `POD_NAME` and `POD_NAMESPACE` environment variables, which have the following association with `tags` in the Tracing configuration:
> 
> ```
> tags['replica'] = <value from POD_NAME>
> tags['namespace'] = <value from POD_NAMESPACE>
> ```
> 
> I wonder if this could solve the problem?

Yeah, it solved the problem

Reply:
has changed `replica` to `instance`